docs/readthedocs/source/doc/UserGuide/k8s.md

```diff
@@ -123,7 +123,17 @@ Instead of lanuching a client container, you can also submit BigDL application f
 
 2. Follow the steps in the [Python User Guide](./python.html#install) to install BigDL in a conda environment.
 
-
+3. Pack conda environment and use on both the driver and executor.
+    - Pack the current conda environment to `environment.tar.gz` (you can use any name you like):
+    ```
+    conda pack -o environment.tar.gz
+    ```
+    - spark-submit with "--archives" and specify python stores for dirver and executor
+    ```
+    --conf spark.pyspark.driver.python=./env/bin/python \
+    --conf spark.pyspark.python=./env/bin/python \ 
```

Comment:
what is .env? shouldn't be .environment?

Reply:
ok 
Yeah, I am planning to add `init_spark_on_k8s_cluster` https://github.com/intel-analytics/BigDL/pull/3698. Then users will use `init_spark_on_k8s_cluster(..,penv_archive="path/environment.tar.gz")`instead of  `--conf spark.pyspark.python=./env/bin/python`
For client mode, init_spark_on_k8s will automatically detect conda and pack to executor pod. 
The doc will update after the two pr merge.

Reply:
okay. better make env and environment align, otherwise it will be confusing.